src/coreclr/src/jit/lclvars.cpp

```diff
@@ -348,6 +348,18 @@ void Compiler::lvaInitArgs(InitVarDscInfo* varDscInfo)
     /* Is there a "this" pointer ? */
     lvaInitThisPtr(varDscInfo);
 
+    unsigned numUserArgsToSkip = 0;
+    unsigned numUserArgs       = info.compMethodInfo->args.numArgs;
```

Comment:
`impInlineInitVars` needs similar changes, but I would ignore it for now and just try to pass all tests with `COMPlus_JitMinOpts=1` so you don't need to deal with inlining and optimizations.

Reply:
Is this true? 

We shouldn't be seeing methods with unmanaged call conventions as inline candidates, so seemingly we wouldn't need any of this new special processing for inlinees.